django_project/frontend/src/store/ticketSlice.ts

```diff
@@ -37,6 +37,15 @@ interface TicketState {
     error: null,
   };
 
+const setCSRFToken = () => {
```

Comment:
Can you put this function in the utils instead? Remember to follow the DRY approach.